### PR TITLE
Add developer profile validation

### LINKED ISSUE #104: Add validation to Developer Profile fields in information structure
Collection name: `"developer-profile"`
Task description in linked Epic issue.
https://github.com/CodersCrew/CodersCard/issues/98#issue-725735917

### PR DESCRIPTION
**Pull request by celinawierzbicka**
Closes #104

### CODE REVIEW

File: static/admin/config.yml [hunk 1]
```diff
@@ -41,24 +41,24 @@ collections:
...
+      - { label: "First name", name: "firstName", widget: "string", pattern: ['.{2,20}', "Must have 2-20 characters"] }
+      - { label: "Last name", name: "lastName", widget: "string", pattern: ['.{2,20}', "Must have 2-20 characters"] }
       - { label: "Avatar image", name: "avatar", widget: "image" }
-      - { label: "Job position", name: "position", widget: "string" }
+      - { label: "Job position", name: "position", widget: "string", pattern: ['.{2,30}', "Must have 2-30 characters"] }
       - { label: "Phone number", name: "phone", widget: "string", required: false }
-      - { label: "Email", name: "email", widget: "string" }
-      - { label: "City of residence", name: "city", widget: "string" }
-      - { label: "Country of residence", name: "country", widget: "string" }
+      - { label: "Email", name: "email", widget: "string", pattern: ['/^.+@.+\..+$/', "Must be a valid e=mail address"] }
```
Comment (mecies) [inline static/admin/config.yml]:
```suggestion
      - { label: "Email", name: "email", widget: "string", pattern: ['/^.+@.+\..+$/', "Must be a valid email address"] }
```
this character is unnecessary 😄

**Review by mecies — CHANGES_REQUESTED** [review]
One small change, other than that looks good to me 😄 

I don't understand the URL regex at all, but I tested it in the browser and it worked for URLs and refused not URLs

File: static/admin/config.yml (hunk 1 above)
Comment (celinawierzbicka) [reply static/admin/config.yml]: ooohh... fixed now :)

**Review by mecies — APPROVED** [review]
Good job, ready to merge ✅

**Pull request merged**